appcues/src/main/java/com/appcues/ui/composables/StepComposition.kt

```diff
@@ -4,69 +4,95 @@ import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.BoxScope
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.PaddingValues
-import androidx.compose.foundation.layout.fillMaxHeight
 import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.rememberScrollState
-import androidx.compose.foundation.verticalScroll
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.platform.LocalDensity
 import com.appcues.data.model.Step
-import com.appcues.trait.StepDecoratingPadding
 import com.appcues.trait.StepDecoratingTrait.StepDecoratingType
+import com.appcues.trait.StickyContentPadding
 import com.appcues.trait.alignStepOverlay
 import com.appcues.ui.primitive.Compose
 
 @Composable
-internal fun Step.ComposeStepContent(
-    index: Int,
-    hasFixedHeight: Boolean,
-    contentPadding: PaddingValues?,
-    stepDecoratingPadding: StepDecoratingPadding
-) {
-    Column(
-        horizontalAlignment = Alignment.CenterHorizontally,
-        modifier = Modifier
```

Comment:
this Modifier block was complex because it was trying to support different configurations based on fixedHeight, content padding,  etc, now we only apply the modifier when composiing the step, everything else is set on the ContentWrapperTrait (modals, tooltip)

Reply:
nice cleanup 👍